spec/presenters/two_factor_auth_code/webauthn_authentication_presenter_spec.rb

```diff
@@ -18,6 +18,21 @@
   let(:phishing_resistant_required) { false }
   let(:platform_authenticator) { false }
 
+  describe '#webauthn_title' do
+    it('shows title for security key mfa') do
+      expect(subject.webauthn_title).to eq(t('titles.present_webauthn'))
+    end
+
+    context('when using platform authenticator mfa') do
+      let(:platform_authenticator) { true }
+      it('shows title for platform authenticator mfa') do
```

Comment:
Nit: For consistency / convention, I wouldn't expect the parentheses on the `it` and `context`

```suggestion
    it 'shows title for security key mfa' do
      expect(subject.webauthn_title).to eq(t('titles.present_webauthn'))
    end

    context 'when using platform authenticator mfa' do
      let(:platform_authenticator) { true }

      it 'shows title for platform authenticator mfa' do
```